### PR TITLE
Update releases md

### PR DESCRIPTION
**Pull request by hansvancalster**
Maintenance PR to reflect new guidelines for admins due to automatisation of a release for a new or updated protocol.

### CODE REVIEW

**Comment by hansvancalster** [comment]
@ElsLommelen feel free to not wait for my return to work to improve this as needed and merge this. Maybe you want to merge this in `renv-setup`. Currently it is set to merge into `main`.

File: RELEASES.md
Comment (ElsLommelen) [inline RELEASES.md:7]:
```suggestion
    1.  This action will update the version number if needed and automatically run these checks:
```

File: RELEASES.md
Comment (ElsLommelen) [inline RELEASES.md:15]:
```suggestion
        In principle, these errors should be addressed by the protocol contributor, but admins can help if needed.
        Note that during this automatic run, a commit to update the version number may be automatically pushed to the remote.
        This should not pose a problem for local clones as this change can be automatically included and will not cause conflicts (i.e. `git pull origin main` with a local protocol branch checked out)
```

File: RELEASES.md
Comment (ElsLommelen) [inline RELEASES.md:44]:
```suggestion
            The rendering is done by `protocolhelper:::render_release()`.
            The same general and protocol-specific tags are added to this commit on `protocols`.
```

**Review by ElsLommelen — COMMENTED** [review]
Some small additions that seemed useful to me to have the whole workflow documented.  (We only recently agreed to also update this version number automatically, so maybe after @hansvancalster made this PR?)

@florisvdh Do you mind checking these changes for spelling as well?  We only merge it to renv-setup, so @hansvancalster can have his final word on the changes there (after trying the workflow himself).

File: RELEASES.md
Comment (ElsLommelen) [inline RELEASES.md:29]:
```suggestion
    To add these commits to your local clone, press pull in the RStudio Git Pane.
```
I suppose also repo-admins don't need this long description to remember they have to pull?  ;-)

File: RELEASES.md
Comment (florisvdh) [inline RELEASES.md:5]:
```suggestion
1.  A GitHub Action will run when a pull request to the main branch is made or updated.
```

File: RELEASES.md
Comment (florisvdh) [inline RELEASES.md:14]:
```suggestion
        Repeat this step until all checks succeed.
```

File: RELEASES.md
Comment (florisvdh) [inline RELEASES.md:17]: Not all admins will use the RStudio buttons to control git.

```suggestion
        If this commit is added in the remote, pull it into your local clone.
```

File: RELEASES.md
Comment (florisvdh) [inline RELEASES.md:19]:
```suggestion
2.  A second GitHub Action will be run when a pull request to be merged into the main branch *is approved*.
```

At least I think this is what is meant.

File: RELEASES.md
Comment (florisvdh) [inline RELEASES.md:31]:
```suggestion
    So pull these commits to get them in your local repo.
```

File: RELEASES.md
Comment (florisvdh) [inline RELEASES.md:27]:
```suggestion
    1.
```

File: RELEASES.md
Comment (florisvdh) [inline RELEASES.md:57]:
```suggestion
    1.  Check success of GitHub releases and publication on Zenodo
```

Github Markdown takes into account the actual number when starting a new numbered list.

**Review by florisvdh — APPROVED** [review]
:+1: Some minor remaining comments below.

**Pull request merged**
